NETWORK/NetworkDoesEntityExistWithNetworkId.md

```diff
@@ -5,11 +5,11 @@ ns: NETWORK
 
 ```c
 // 0x18A47D074708FD68 0x1E2E3177
-BOOL NETWORK_DOES_ENTITY_EXIST_WITH_NETWORK_ID(Entity entity);
+BOOL NETWORK_DOES_ENTITY_EXIST_WITH_NETWORK_ID(int netID);
 ```
 
-
+Gets wether the network ID exists and if this belongs to a game object.
 ## Parameters
-* **entity**: 
+* **netId**: 
```

Comment:
`netId` vs. `netID` in argument leads to build failure.

Reply:
Didn't realise at the time that these files were being used for building.
Did realise it later on while trying to create a self made native.

I've fixed the issue